Jellyfin.Server.Implementations/Users/DeviceAccessEntryPoint.cs

```diff
@@ -38,27 +36,27 @@ public void Dispose()
         {
         }
 
-        private void OnUserUpdated(object? sender, GenericEventArgs<User> e)
+        private async void OnUserUpdated(object? sender, GenericEventArgs<User> e)
```

Comment:
This used to be synchronous unless I'm mistaken. But now it's asynchronous and I _think_ that the eventhandler is synchronous, so does it wait for execution? Do we need to use `AsyncCompletedEventArgs` instead?

Reply:
I'm not certain, both sync and async events are scattered through the code and as far as I can tell none of them use `AsyncCompletedEventArgs`